docs/content/docs/vagrant__introduction.md

```diff
@@ -16,6 +16,8 @@ $ git clone https://github.com/Kitware/HPCCloud-deploy.git
 $ git clone https://github.com/Kitware/cumulus.git
 $ git clone https://github.com/Kitware/girder.git
 
+$ ln -s HPCCloud hpccloud
```

Comment:
I was thinking of it, but I liked the consistency of the git clone commands.
Also I feel it highlight the need to have HPCCloud in lower case in order to have the DEVELOPMENT=1 environment properly working.

But if you feel I should change it, I can, I'm just worried that people will miss it.